content/en-us/reference/engine/enums/AudioApiRollout.yaml

```
@@ -1,23 +1,25 @@
 name: AudioApiRollout
 type: enum
-summary: ''
-description: ''
+summary: |
+  Used to determine whether voice chat uses the Audio API
+description: |
+  Used to determine whether voice chat uses the Audio API
 code_samples: []
 tags: []
 deprecation_message: ''
 items:
   - name: Disabled
-    summary: ''
+    summary: 'Voice chat will not use the Audio API, meaning its implementation cannot be controlled.'
```

I think we should word this more neutrally, maybe something like:
```suggestion
    summary: 'Voice chat will not use the Audio API, meaning its implementation is completely automatic and hidden.'
```